packages/core/src/components/templates/SessionListCard/SessionListCard.js

```diff
@@ -0,0 +1,363 @@
+                </Animated.View>
+              )}
+            </View>
+          );
+        })}
+      </ScrollView>
+    </View>
+  );
+};
+
+const getStyles = Colors =>
+  StyleSheet.create({
+    container: {
+      maxHeight: Dimensions.get('window').height * 0.45,
+    },
+    listContainer: {
+      flexDirection: 'row',
+      alignItems: 'center',
+    },
+    scrollView: {
+      width: '100%',
+    },
+    contentContainer: {
+      alignItems: 'center',
+      paddingHorizontal: '5%',
+    },
+    cardContainer: {
+      flexDirection: 'row',
+      marginVertical: 4,
+      width: '100%',
+      alignItems: 'center',
+      height: Dimensions.get('window').height * 0.15,
+    },
+    borderCard: {
+      borderWidth: 1,
+      borderColor: Colors.secondaryColor_dark.background,
+    },
+    selectBorderCard: {
+      borderWidth: 1,
+      borderColor: Colors.infoColor.background,
+    },
+    borderIsDefaultCard: {
+      borderWidth: 1,
+      borderColor: Colors.primaryColor.background,
+    },
+    textTitle: {
+      fontSize: 18,
+      fontWeight: 'bold',
+    },
+    textContainer: {
+      flexDirection: 'column',
+      width: '70%',
+      marginLeft: 20,
+    },
+    imageContainer: {
+      width: '20%',
+      height: 100,
+    },
+    iconContainer: {
+      alignSelf: 'flex-end',
+      flexDirection: 'row',
+      paddingHorizontal: '5%',
+    },
+    icon: {
+      marginHorizontal: 4,
+    },
+    details: {
+      fontStyle: 'italic',
+    },
+    iconPlus: {
+      marginHorizontal: 4,
+      alignSelf: 'center',
+      justifyContent: 'center',
+      alignItems: 'center',
+      backgroundColor: Colors.secondaryColor_dark.foreground,
+      borderWidth: 2,
+      borderColor: Colors.primaryColor.background,
+      borderRadius: Dimensions.get('window').width * 0.07,
+      width: Dimensions.get('window').width * 0.07,
+      height: Dimensions.get('window').width * 0.07,
+    },
+    squareIconContainer: {
+      flexDirection: 'column',
+      position: 'absolute',
+      right: -Dimensions.get('window').height * 0.07 - 10,
+    },
+    iconTrash: {
+      marginHorizontal: 4,
+      alignSelf: 'center',
+      justifyContent: 'center',
+      alignItems: 'center',
+      backgroundColor: Colors.secondaryColor_dark.foreground,
+      borderWidth: 2,
+      borderColor: Colors.errorColor.background,
+      borderRadius: 10,
+      width: Dimensions.get('window').height * 0.07,
+      height: Dimensions.get('window').height * 0.07,
+      marginVertical: 3,
+    },
+    iconPen: {
+      marginHorizontal: 4,
+      alignSelf: 'center',
+      justifyContent: 'center',
+      alignItems: 'center',
+      backgroundColor: Colors.secondaryColor_dark.foreground,
+      borderWidth: 2,
+      borderColor: Colors.infoColor.background,
+      borderRadius: 10,
+      width: Dimensions.get('window').height * 0.07,
+      height: Dimensions.get('window').height * 0.07,
+      marginVertical: 3,
+    },
```

Comment:
That's a lot of similar style, please create sub component which contains basic style and to which you can give border color, icon name, on press etc to simplify